task/sonarqube-scanner/0.1/sonarqube-scanner.yaml

```diff
@@ -62,10 +73,16 @@ spec:
           if [[ -n "$(params.SONAR_PROJECT_KEY)" ]]; then
             replaceValues $(workspaces.source-dir.path)/sonar-project.properties sonar.projectKey $(params.SONAR_PROJECT_KEY)
           fi
+          if [[ -n "$(params.SONAR_LOGIN)" ]]; then
+            replaceValues $(workspaces.source-dir.path)/sonar-project.properties sonar.login $(params.SONAR_LOGIN)
+          fi
         else
           touch sonar-project.properties
           echo "sonar.projectKey=$(params.SONAR_PROJECT_KEY)" >> sonar-project.properties
           echo "sonar.host.url=$(params.SONAR_HOST_URL)" >> sonar-project.properties
+          if [[ -n "$(params.SONAR_PROJECT_KEY)" ]]; then
```

Comment:
I guess the check here should be for `SONAR_LOGIN`

Reply:
😅 fixed